app/package.json

```diff
@@ -91,6 +92,7 @@
     "reset-css": "2.2.1",
     "semver-diff": "2.1.0",
     "simplemde": "1.11.2",
+    "slnodejs-core": "^4.0.286",
```

Comment:
OK, I don't exactly know why that was added to the production section it seems useless. So I just deleted this.

Reply:
Regarding to '-E' flag I would prefer to leave this as is, since they update their agent constantly and it might be incompatible with newer version of their browser extension.

Reply:
And a command to build with sealights is: 'npm run sealights' but it's necessary to run some commands before to create instrumented version of UI-code.